pkg/kritis/cryptolib/pgp.go

```diff
@@ -18,13 +18,18 @@ package cryptolib
 
 import (
 	"bytes"
+	"fmt"
 	"io/ioutil"
 
 	"github.com/pkg/errors"
 	"golang.org/x/crypto/openpgp"
 	"golang.org/x/crypto/openpgp/armor"
 )
 
+//**************
```

Comment:
Remove the block comment. They are not common in go and it's pretty clear what pgpVerifier and pgpSigner do by the type name

Reply:
Done.